vendor/cmdliner/src/cmdliner_manpage.ml

```diff
@@ -46,7 +45,7 @@ let s_see_also = "SEE ALSO"
 let s_created = ""
 let order =
   [| s_name; s_synopsis; s_description; s_created; s_commands;
-     s_command_aliases; s_arguments; s_options; s_common_options; s_exit_status;
```

Comment:
IIRC, this was a local patch we had in Dune. @shonfeder does that ring a bell?

Has someone tried to upstream this patch?

Reply:
Yeah, that's right. Not sure if this was the right way to add in the section. But that's all it does, so not sure if this is worth upstreaming, or if we need it.

Reply:
Good catch. I wasn't aware. In the future, if we make modifications like this we should do the following:

* Fork cmdliner into `ocaml-dune`
* Apply the patch there.
* Change the update script to point to our fork

This will make it so that we don't accidentally lose patches.